packages/office-ui-fabric-react/src/components/Announced/examples/Announced.BulkOperations.Example.tsx

```diff
@@ -38,143 +50,116 @@ const _names: string[] = [
-  modified: string;
-  modifiedby: string;
-  filesize: string;
-}
+};
 
-export interface IAnnouncedBulkOperationsExampleState {
-  items: IFileExampleItem[];
-  numberOfItems: number;
-}
-
-export class AnnouncedBulkOperationsExample extends React.Component<{}, IAnnouncedBulkOperationsExampleState> {
-  private _selection: Selection;
-  private _dragDropEvents: IDragDropEvents;
-  private _draggedItem: IFileExampleItem | undefined;
-  private _draggedIndex: number;
-
-  constructor(props: {}) {
-    super(props);
-
-    this._selection = new Selection();
-    this._dragDropEvents = this._getDragDropEvents();
-    this._draggedIndex = -1;
-
-    if (_items.length === 0) {
-      for (let i = 0; i < 20; i++) {
-        _items.push({
-          key: 'item-' + i,
-          name: 'Item ' + i,
-          modified: getMockDateString(),
-          modifiedby: _names[Math.floor(Math.random() * _names.length)],
-          filesize: Math.floor(Math.random() * 30).toString() + ' MB',
-        });
-      }
-    }
-
-    this.state = {
-      items: _items,
-      numberOfItems: 0,
-    };
-  }
+const onItemInvoked = (item: IFileExampleItem): void => {
+  alert(`Item invoked: ${item.name}`);
+};
 
-  public render(): JSX.Element {
-    const { items } = this.state;
-    const stackTokens: IStackTokens = { childrenGap: 10 };
-
-    return (
-      <Stack tokens={stackTokens}>
-        <Text>Turn on Narrator and drag and drop the items.</Text>
-        <Text>
-          Note: This example is to showcase the concept of copying, uploading, or moving many items and not fully
-          illustrative of the real world scenario.
-        </Text>
-        {this._renderAnnounced()}
-        <MarqueeSelection selection={this._selection}>
-          <DetailsList
-            setKey="items"
-            items={items}
-            columns={_columns}
-            selection={this._selection}
-            selectionPreservedOnEmptyClick={true}
-            onItemInvoked={this._onItemInvoked}
-            onRenderItemColumn={this._onRenderItemColumn}
-            dragDropEvents={this._dragDropEvents}
-            ariaLabelForSelectionColumn="Toggle selection"
-            ariaLabelForSelectAllCheckbox="Toggle selection for all items"
-          />
-        </MarqueeSelection>
-      </Stack>
-    );
+const onRenderItemColumn = (item: IFileExampleItem, index: number, column: IColumn): React.ReactNode => {
+  if (column.key === 'name') {
+    return <Link data-selection-invoke>{item[column.key]}</Link>;
   }
 
-  private _renderAnnounced(): JSX.Element | undefined {
-    const { numberOfItems } = this.state;
-    if (numberOfItems > 0) {
-      return (
-        <Announced message={`${numberOfItems} item${numberOfItems === 1 ? '' : 's'} moved`} aria-live={'assertive'} />
-      );
+  return item[column.key as keyof IFileExampleItem];
+};
+
+const stackTokens: IStackTokens = { childrenGap: 10 };
+
+export const AnnouncedBulkOperationsExample: React.FunctionComponent = () => {
```

Comment:
I started thinking about this example while testing it with Narrator and realized that it doesn't even make much sense...pretty sure a real Narrator user would not typically be moving things with drag/drop!! Ideally we should change this to demonstrate deleting multiple items or something like that (and get rid of the MarqueeSelection for simplicity), but it would also be okay to just leave it for now since this isn't a new problem.